src/panels/lovelace/cards/hui-shopping-list-card.ts

```diff
@@ -398,7 +398,8 @@ class HuiShoppingListCard extends SubscribeMixin(LitElement)
       }
 
       .checked span {
-        color: var(--primary-color);
+        color: var(--primary-text-color);
```

Comment:
Should we add a variable for those headers? So people can theme them?

Reply:
I will do that in a separate PR 👍.